ibm_catalog.json

```diff
@@ -0,0 +1,379 @@
+                            "key": "provider_visibility",
+                            "hidden": true,
+                            "options": [
+                                {
+                                    "displayname": "private",
+                                    "value": "private"
+                                },
+                                {
+                                    "displayname": "public",
+                                    "value": "public"
+                                },
+                                {
+                                    "displayname": "public-and-private",
+                                    "value": "public-and-private"
+                                }
+                            ]
+                        }
+                    ],
+                    "architecture": {
+                        "features": [
+                            {
+                                "title": " ",
+                                "description": "Configured to use IBM secure by default standards, but can be edited to fit your use case."
+                            }
+                        ],
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "Private Path",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-vpc-private-path-external-connectivity/main/reference-architecture/reference-architectures/private-path-external-connectivity.svg",
+                                    "type": "image/svg+xml"
+                                },
+                                "description": "You can deploy a <b>Private Path</b> service on IBM Cloud to facilitate secure, private connectivity between IBM Cloud Virtual Private Cloud (VPC) environments and on-premises networks or external services using this solution. This solution automates the deployment of IBM Cloud's Private Path and Application Load Balancer (ALB) services to enable enterprises to establish robust, private, and scalable network connections.<br><br>The Private Path service establishes private Layer 3 network connectivity between IBM Cloud VPC resources and external destinations such as on-premises data centers or other cloud environments. It leverages secure IPsec tunnels and Direct Link connections to maintain encrypted, low-latency communication, ensuring sensitive workloads can operate within a highly secure, isolated network path that bypasses the public internet.<br><br>By automating the provisioning and configuration of these components, the solution delivers a seamless, production-ready framework for establishing hybrid connectivity, optimizing both performance and security. This approach enhances operational agility while maintaining enterprise-grade standards for secure data exchange across cloud and on-premises boundaries."
```

Comment:
Here we are only talking about Private path service. Addons information is missing.